Bottom: d01783e69f8ae9c8a1e0dd7e92f6fbb422ff9ca4
Top:    97cf4936a08b14598059c966c3173377b21dab87
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2013-12-31 20:52:37 -0500

Refresh of zarlink

---

diff --git a/target/linux/ramips/dts/rt5350.dtsi b/target/linux/ramips/dts/rt5350.dtsi
index 5487669..01de1d7 100644
--- a/target/linux/ramips/dts/rt5350.dtsi
+++ b/target/linux/ramips/dts/rt5350.dtsi
@@ -212,7 +212,7 @@
 
 			interrupt-parent = <&intc>;
 			interrupts = <4>;
-		}
+		};
 
 		gdma@2800 {
 			compatible = "ralink,rt5350-gdma", "ralink,rt2880-gdma";
@@ -220,7 +220,7 @@
 
 			interrupt-parent = <&intc>;
 			interrupts = <7>;
-		}
+		};
 	};
 
 	pinctrl {
